rust/gui-client/src-tauri/src/client/gui.rs

```diff
@@ -557,6 +557,7 @@ impl Controller {
             None,
             callback_handler.clone(),
             Some(MAX_PARTITION_TIME),
+            tokio::runtime::Handle::current(),
```

Comment:
This should work since this function is called from a Tokio context, but note that the smoke tests don't cover this yet because they don't log in to Firezone.

Reply:
Yeah I just did the quickest fix possible here. I think it would be cleaner to pass a `Handle` down the stack from a place where we definitely know that we have a runtime, i.e. `main`.